.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
We used to use prettier, there was debate, we decided to switch to clang format, since then we moved to TypeScript and now things are a bit up in the air. Am ok checking this in but we should revisit and centralize sometime post-M1.